build/crds/build-crds.sh

```diff
@@ -67,18 +67,18 @@ EOF
 }
 
 build_helm_resources() {
-  # Add helm annotations to all CRDS
-  "$YQ_BIN_PATH" w -i -d'*' "$HELM_CRDS_FILE_PATH" "metadata.annotations[helm.sh/resource-policy]" keep
-  
-  # add header
-  sed -i '1s/^/{{- if semverCompare ">=1.16.0" .Capabilities.KubeVersion.GitVersion }}\n/' "$HELM_CRDS_FILE_PATH"
-  sed -i '1s/^/{{- if .Values.crds.enabled }}\n/' "$HELM_CRDS_FILE_PATH"
-  
   echo "Generating helm resources.yaml"
   {
+    # add header
+    echo "{{- if .Values.crds.enabled }}"
+    echo "{{- if semverCompare \">=1.16.0\" .Capabilities.KubeVersion.GitVersion }}"
+
+    # Add helm annotations to all CRDS and skip the first 4 lines of crds.yaml
+    "$YQ_BIN_PATH" w -d'*' "$CRDS_FILE_PATH" "metadata.annotations[helm.sh/resource-policy]" keep | tail -n +5
```

Comment:
Why is `$HELM_CRDS_FILE_PATH` gone?

Reply:
It's cut off in the diff view, but see line 88 where everything in this block is piped to `>>"$HELM_CRDS_FILE_PATH"`.

Reply:
Actually, the magic is in the removal of the `-i` flag, missed it in the first review.

Reply:
Right, and this command also takes `$CRDS_FILE_PATH` as the input